ckan/cli/search_index.py

```diff
@@ -77,6 +77,43 @@ def clear(dataset_name: str):
         clear_all()
 
 
+def list_orphans(return_list: bool=False) -> list|None:
+    import ckan.logic as logic
```

Comment:
Can you import it on the top-level of module?

Reply:
Imports at module top level now.